eo-runtime/src/test/eo/org/eolang/collections/list-tests.eo

```diff
@@ -354,3 +354,30 @@
         list
           args.
             b 1 2 "fail"
+
+[] > reduce-with-memory-test
```

Comment:
@EugeneDar would be better to give tests more descriptive names. We know that it's a test, no need to repeat this. It's better to say what the object under test must do, for example: `reduce-can-work-memory`. see: https://stackoverflow.com/questions/96297/what-are-some-popular-naming-conventions-for-unit-tests

Reply:
@yegor256 ok, I will fix it in eo-collections.